code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -313,6 +170,10 @@ var/list/gaslist_cache = init_gaslist_cache()
 	//Compares sample to self to see if within acceptable ranges that group processing may be enabled
 	//Returns: a string indicating what check failed, or "" if check passes
 
+/datum/gas_mixture/proc/react(turf/open/dump_location)

```

Comment:
\>nobody actually played with my version to profile it

protip: the difference basically goes away

its the last call bullshit in byond